src/scss/vendors/_vendors.scss

```diff
@@ -31,6 +31,45 @@
 	}
 }
 
+// uio constrast focus and hover
+@each $theme, $foregroundColor, $backgroundColor in $uio-themes {
+	.gpii-ext-theme-#{$theme},
+	.fl-theme-#{$theme} {
+		@include uio-focus-style($foregroundColor, $backgroundColor);
+
+		#defaultContainer button {
```

Comment:
The common css at line 40-70 that doesn't need to consume `$foregroundColor` and `$backgroundColor` should be outside of the loop and under a generic selector `[class^="gpii-ext-theme"], [class^="fl-theme"]:not(.fl-theme-prefsEditor-default)`, the same one at line 74, 75. The problem with having them in the loop is these line will be generated repeatedly for each theme. With 8 themes UIO has, the generated css file for this chunk will be 7 times bigger.

Only css that needs `$foregroundColor` and `$backgroundColor` variables goes into the loop.